meson.build

```diff
@@ -283,6 +283,18 @@ if not get_option('mqtt-support').disabled()
   endif
 endif
 
+# deepview-rt
+deepview_rt_dep = dependency('', required: false)
+if not get_option('deepview-rt-support').disabled()
+  deepview_rt_dep = dependency('deepview-rt', required: false)
+  if (not deepview_rt_dep.found())
+    deepview_rt_lib = cxx.find_library('deepview-rt', required: false)
+    if (deepview_rt_lib.found() and cxx.check_header('deepview_rt.h'))
+      deepview_rt_dep = declare_dependency(dependencies: [ deepview_rt_lib ])
+    endif
+  endif
+endif
```

Comment:
You need to generate errors if ```deepview-rt-support``` is force enabled by users (```-Ddeepview-rt-support=enabled```). It appears that it will act as if it is ```auto```.